options/test/HiFaceGAN/test_hifacegan.yml

```diff
@@ -0,0 +1,49 @@
+name: HiFaceGAN_SR4x_test
+model_type: SRModel
```

Comment:
You may inherit SRModel to build a new class, for changing the train/val behaviors mentioned above.